Dockerfile

```diff
@@ -0,0 +1,30 @@
+FROM debian:stretch-slim
+
+ARG librdkafka_version=v0.11.1-RC1
+ARG yajl_version=2.1.0
+
+COPY . /usr/src/kafkacat
+
+RUN set -ex; \
+  runtimeDeps='zlib1g liblz4-1 libssl1.1 libsasl2-2'; \
+  buildDeps='curl ca-certificates build-essential zlib1g-dev liblz4-dev libssl-dev libsasl2-dev python cmake'; \
+  export DEBIAN_FRONTEND=noninteractive; \
+  apt-get update && apt-get install -y $runtimeDeps $buildDeps --no-install-recommends; \
+  rm -rf /var/lib/apt/lists/*; \
+  \
+  cd /usr/src/kafkacat; \
+  \
+  sed -i "s|github_download \"edenhill/librdkafka\" \"master\"|github_download \"edenhill/librdkafka\" \"${librdkafka_version}\"|" ./bootstrap.sh; \
+  sed -i "s|github_download \"lloyd/yajl\" \"master\"|github_download \"lloyd/yajl\" \"${yajl_version}\"|" ./bootstrap.sh; \
+  \
+  echo "Source versions:"; \
+  grep ^github_download ./bootstrap.sh; \
+  \
+  ./bootstrap.sh; \
+  mv ./kafkacat /usr/local/bin/; \
+  \
+  rm -rf /usr/src/kafkacat/tmp-bootstrap; \
+  apt-get purge -y --auto-remove $buildDeps; \
+  rm /var/log/dpkg.log /var/log/alternatives.log /var/log/apt/*.log
+
```

Comment:
Run `kafkacat -V` to verify that the binary still works after package removal.

Reply:
It bailed on lack of libcrypto. I've added all libs now. The resulting layer totals 10.2 MB now instead of 6.6 prior to building with libssl+libsasl.

Reply:
Aye, add `kafkacat -V` as a final step which makes sure all required dependencies are in place.

Reply:
ac6c085